source/includes/_queries.md

```diff
@@ -40,13 +41,22 @@ Each query consists of following things:
     <li> **Query object** might be any of types, described below </li>
 </ul>
 
-**Signatures** contain one or many signatures (ed25519 pubkey + signature):
+**Signatures** contain one or many signatures (ed25519 pubkey + signature).
+
+## Validation
+
+The validation for all commands includes:
+
+ * timestamp — it should be in more than time of the peer minus 24 hours
+ * signature of query creator — is used for checking the identity of query creator
+ * query counter — is checked to be incremented with every subsequent query from this query creator
+ * roles — depending on the role of query creator, it can be applied to the same account, account in the domain, in the whole system or not allowed at all
 
 ## Get account
 
 ### Purpose
 
-Purpose of _get account_ query is to get state of an account.
+Purpose of _get account_ query is to get the state of an account.
 
 Given this Query, in successful case Iroha returns `AccountResponse`, which contains Account object with following fields:
```

Comment:
Considering we do not cover successful case, "in successful case" is not needed. Also "Iroha returns" does not sound very good to me. As the query itself is an action, we can say "This Query returns ..." which is shorter.